apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -766,6 +766,38 @@ contexts.forEach((context: Context) => {
       expect(messageBodies).to.include('Hello there 2');
     });
 
+    it(`should deliver message if the Workflow Definition doesn't contain preferences [${context.name}]`, async () => {
+      process.env.IS_WORKFLOW_PREFERENCES_ENABLED = 'true';
+      const workflowId = `without-preferences-workflow-${`${context.name}-${uuidv4()}`}`;
+      const newWorkflow = workflow(workflowId, async ({ step }) => {
+        await step.inApp('send-in-app', () => ({ body: 'Hello there 1' }));
+      });
+
+      /*
+       * Delete `preferences` from the Workflow Definition to simulate an old
+       * Workflow Definition (i.e. from old Framework version) that doesn't have the `preferences` property.
+       */
+      delete newWorkflow.definition.preferences;
```

Comment:
we need in because the `workflow` is defined with preferences by default, right?

Reply:
Yes, that's correct. E2E suite always using the latest Framework version which produces a `preferences` property, so we need to delete it manually.